src/d3a/models/area/event_dispatcher.py

```diff
@@ -152,10 +152,18 @@ def create_area_agents(self, is_spot_market, market):
                 return
             # Only connect an InterAreaAgent if we have a parent, a corresponding
             # timeframe market exists in the parent and we have no strategy
+            if ConstSettings.GeneralSettings.EVENT_DISPATCHING_VIA_REDIS:
```

Comment:
Oh, sorry, this was still in there. But we decided against just parsing the market_ids, because we said that the area and IAAS are always connected to the markets and we deal with the communication between market and area later (when deploying). Now I am confused why you let it in.